### ISSUE TITLE
Modify prompt setting

### ISSUE THREAD

**Issue by corbindavenport** [corbindavenport/alt-text-creator#2]
There should be a setting for changing the prompt used by the GPT-4 API call. This could be useful if the Alt text needs to be longer or changed in some other way.
